src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -857,14 +864,16 @@ class Diagram extends Component<Props> {
   }
 
   render() {
-    const { node, index } = this.state.editingNodeItem || {}
-    const formType: string = node?.nodeType || node?.type
+    const { node, index, data } = this.state.editingNodeItem || {}
+    const formType: string = node?.nodeType || node?.type || this.state.editingNodeItem?.type
 
     let editingNodeItem
     if (formType === 'say_something') {
       editingNodeItem = node?.contents?.[index]
     } else if (formType === 'trigger') {
       editingNodeItem = node?.conditions?.[index]
+    } else if (formType === 'entity') {
+      editingNodeItem = data
```

Comment:
Should we rename this var as variables (entities) are not nodes

Reply:
That's why I added the "data" property, i'll rename it "currentItem"